pom.xml

```diff
@@ -897,14 +897,23 @@
           <version>3.16.0</version>
```

Comment:
Version 3.20.0 is available. I can update it with this pr or wait for a follow-on. I just need to know if it is available.